♻️  Replace ContainerAwareCommand usage by DI (#490)
sonata-project/SonataNotificationBundle: UPGRADE-4.0.md

### Configuration

* `sonata_notification.admin.enabled` is not enabled by default

### Commands

> They no longer extends the deprecated `ContainerAwareCommand` class. They extend `Command` class now.

Reviewer: ```suggestion
All commands now extend `Symfony\Component\Console\Command\Command` class instead of the deprecated `Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand` class.
```
